link/src/main/java/com/stripe/android/link/LinkPaymentLauncher.kt

```diff
@@ -239,7 +239,7 @@ class LinkPaymentLauncher @AssistedInject internal constructor(
 
     @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
     companion object {
-        val LINK_ENABLED = BuildConfig.DEBUG
+        val LINK_ENABLED = false
```

Comment:
Had to revert my earlier change because this breaks the `PaymentOptionsActivityTest`.